project-koku/koku
# Enable masu ingest/processing in koku

Addresses #888 #889

Enabling masu ingest and processing logic in Koku repo.

**Testing**
1. Generate Nise OCP payload
2. Start Koku and connect koku-listener to upload service network
3.  Add OCP Provider
4. Upload OCP payload to upload service.
5. Verify OCP processing is complete.
6.  Add AWS provider.
7. Ingest data with masu-server /download
8. Verify AWS processing is complete.
